extensions/transform/org.eclipse.smarthome.transform.exec/README.md

```diff
@@ -6,14 +6,41 @@ The external program must either be in the executable search path of the server
 
 ## Example
 
-With the command line 
+**.items**
 
+This will replace the visible label in the UI with the transformation you apply with the command <YourCommand>.
+```java
+String yourItem "Some info  [EXEC(/absolute/path/to/your/TransformProgram %s):]"
 ```
+  
+**.thing**
+
+This example shows the transformation of the output channel of an item.
+```java
+Thing yourThing [ ... , transform="EXEC(/absolute/path/to/your/TransformProgram %s)"]
```

Comment:
I am not aware of any such feature and would claim that this whole "things" section should be removed from here.

Reply:
But it works and makes a lot nice things possible. 

https://community.openhab.org/t/2-openhab2-rpi-system-temperature-chart-with-persistence/35182

```php
Thing exec:command:cpuTemp [
        // Command to execute
        command="cat /sys/class/thermal/thermal_zone0/temp",
        // interval time in seconds
        interval=10,
        // should it run when input channel is changed
        autorun=false,
        // EXEC transformation to make 44007/1000 => 44.007
        // Using python to calculate
        transform="EXEC(python3 -c \"print(%s/1000)\")"]
```

Reply:
But as @sjka mentioned above, the Exec binding does not exist in ESH, so you should not refer to it here. Such documentation is much better off in the Exec binding docs itself.

Reply:
I do not refer to the exec Binding anymore, just to the transformation. Only the last three lines from the code above are relevant. It is embedded in a example to show how you may use it but schould be valid for all things.

Reply:
You do refer to the exec binding, because the transformation parameter only exists for that and does not make any sense otherwise.

Reply:
OK. Please elaborate as a do not understand.

I add some information to `org.eclipse.smarthome.transform.exec/README.md` which is  transfomartion based on the `openhab-transformation-exec` this can be used as a transformation service, same as the JSON, REGEX, JAVAScript, ... 
I use it as i would use a REGEX transformation. 

The EXEC Binding does not have "%s" it supports "%1$tY-%1$tm-%1$td" and "%2$s". To my limited knowledge.

This transformation can be used with things as it is done 
```
http="<[http://example.com/var/value:60000:EXEC(/absolute/path/to/my/program %s)]" }
```
in the openhab 1 wiki.
https://github.com/openhab/openhab1-addons/wiki/Transformations

Reply:
@Josar What I am talking about is the `transform=` parameter in this example. This does not exist for "Things", it only exists for the Exec binding (not being part of ESH) and thus should be documented in that binding and not here.

If we can agree to remove the "*.thing" example from here, it should imho be ready to be merged.

Reply:
@kaikreuzer OK i thought the transformation is generally availabel for things I. Was not aware that it is not. Is there a reason why the transformation is not a generic feature of all things? I can imagine it could be usefull.

Reply:
I agree that it makes sense for all things - and that's why it should not be a feature that every binding needs to support in their things (parameters are always specific to the binding and need to be handled by them).
My plan would be to have a "transformation profile" (see [the new profiles](https://www.eclipse.org/smarthome/documentation/concepts/profiles.html)), which could then be hooked on the link between any channel and the item and do their magic there.

Reply:
I expected that the transformation prarameter would be inherited to all thing classes. Which sound a lot simpler than profiles.

I read the link and think i understand the words written but i can not imagine how i would define a profile. Besides that it sounds like there will be a additional `.profiles` file which i will have to edit.